components/instructions/programs/uxdProtocol.tsx

```diff
@@ -0,0 +1,102 @@
+import { Connection } from '@solana/web3.js'
+import { struct, u8, u48 } from 'buffer-layout'
+import { AccountMetaData } from '../../../models/accounts'
+
+const UXD_INIT_CONTROLLER_INSTRUCTION = {
+  FXY9qRsCxg6ioEFLu9ECQ5v2YF2qjeE1Tcnin3DUBW76: {
+    1: {
+      name: 'UXD - Initialize Controller',
```

Comment:
(this one is still initialize controller though, Controller being one of the pieces of the UXDProgram)